tox.ini

```diff
@@ -43,7 +43,8 @@ commands =
 passenv = HOME
 basepython = python3
 changedir = ./examples/
-deps = mechanical-markdown
+deps =
+    mechanical-markdown
 commands =
     ./validate.sh metadata
```

Comment:
```suggestion
    ./validate.sh metadata
    ./validate.sh error_handling
```

Reply:
You had resolved my comment here without actually doing the thing I told you to do :) Otherwise you would have seen that this test is failing!